recipes/openvino/all/conanfile.py

```diff
@@ -62,7 +63,8 @@ class OpenvinoConan(ConanFile):
         "enable_tf_frontend": True,
         "enable_tf_lite_frontend": True,
         "enable_paddle_frontend": True,
-        "enable_pytorch_frontend": True
+        "enable_pytorch_frontend": True,
+        "enable_jax_frontend": True
```

Comment:
Please, add condition to remove enable_jax_frontend in case the built version is < 2024.3.0:

```python
def config_options(self):
    if Version(self.version) < "2024.3.0":
        del self.options.enable_jax_frontend
```